cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -87,6 +87,10 @@ function _registry_volume() {
 function _add_common_params() {
     # shellcheck disable=SC2155
     local params="--nodes ${KUBEVIRT_NUM_NODES} --memory ${KUBEVIRT_MEMORY_SIZE} --cpu 6 --secondary-nics ${KUBEVIRT_NUM_SECONDARY_NICS} --random-ports --background --prefix $provider_prefix ${KUBEVIRT_PROVIDER} ${KUBEVIRT_PROVIDER_EXTRA_ARGS}"
+
+    dns_host_port=53
```

Comment:
This can be problematic for stuff like NetorkManager with dnsmasq where the dns server is running at 127.0.0.1

Reply:
on fedora for example the dns service runs on 127.0.0.53

do we have a real example where it is problematic ? can we first meet that problem and then fix if needed ?
it will be need to only parameterize this parameter so the user can supply it
(and change tests that we will present with the new feature with `nslookup -p`, assuming this way we are going to test)

Reply:
I had to edit my network settings because I was running dnsmasq locally already

Reply:
PR to fix that
https://github.com/kubevirt/kubevirtci/pull/903